actioncable/lib/action_cable/engine.rb

```diff
@@ -60,7 +60,7 @@ class Engine < Rails::Engine # :nodoc:
     initializer "action_cable.set_work_hooks" do |app|
       ActiveSupport.on_load(:action_cable) do
         ActionCable::Server::Worker.set_callback :work, :around, prepend: true do |_, inner|
-          app.executor.wrap do
+          app.executor.wrap(source: "unhandled_error.action_cable") do
```

Comment:
To Sentry, the `unhandled_error` prefix is not necessary as we'd [add another tag called `handled`](https://github.com/getsentry/sentry-ruby/blob/8bd4054ed2f27b9fb571cd92c409ab10cc3cbd3c/sentry-rails/lib/sentry/rails/error_subscriber.rb#L7) to the event. So users can use `handled` + `source` tag to do the searching.

Reply:
Right, in this case the `unhandled_error` is to signal that the application, not the framework, let something bubble up.

There might be a better name for this.

Reply:
What do you think if we set the naming rules like `[app.][feature.]<component>`. For example:

- `app.action_cable` - user's `action_cable` logic
- `action_cable` - `action_cable` internal (if possible)
- `redis_cache_store.active_support`
- `app.runner.railties` - user's runner code


I try to mimic the same pattern as the `ActiveSupport` instrumentation has. But at here the `<function>` is not always available. And we need an extra flag to indicate if it's the user's code.

Reply:
I'd likely go with `application` over `app`, it's longer, but Rails doesn't use that much. But yes I like the general idea.